src/api/api.js

```diff
@@ -115,7 +115,9 @@ class Api {
     const valid = await this.validator.verify(timedArray, req.params.signature);
     res.json({
       status: 'OK',
```

Comment:
One idea for the future may be to have simple helper functions that generate an `Error` and `OK` response using the appropriate templates.  That would prevent the literal status strings from being duplicated throughout the code and would prevent not using the `errors` and `result` properties in the respective objects.

Reply:
True.


Once I finished writing tests, you are free to enhance the code as you wish 😉